bindings/ergo-lib-wasm/src/wallet.rs

```diff
@@ -62,6 +68,42 @@ impl Wallet {
             .map(Transaction::from)
     }
 
+    /// Sign a transaction:
+    /// `tx` - transaction to sign
+    /// `boxes_to_spend` - boxes corresponding to [`UnsignedTransaction::inputs`]
+    /// `data_boxes` - boxes corresponding to [`UnsignedTransaction::data_inputs`]
+    #[wasm_bindgen]
+    pub fn sign_transaction_multi(
+        &self,
+        _state_context: &ErgoStateContext,
+        tx: &UnsignedTransaction,
+        boxes_to_spend: &ErgoBoxes,
+        data_boxes: &ErgoBoxes,
+        hints: &HintsBag,
+    ) -> Result<Transaction, JsValue> {
+        let boxes_to_spend = TxIoVec::from_vec(boxes_to_spend.clone().into()).map_err(to_js)?;
+        let mut tx_hints = TransactionHintsBag::empty();
+        tx_hints.add_hints_for_input(0, hints);
```

Comment:
Why does it handle hints for the first input only?

Reply:
I Merged the multi_sig_signer with the signer and solved this issue.